back/lib/participation_method/ideation.rb

```diff
@@ -15,7 +15,8 @@ def allowed_ideas_orders
     end
 
     def assign_defaults(input)
-      input_status_code = input.creation_phase&.prescreening_enabled ? 'prescreening' : 'proposed'
+      current_phase = TimelineService.new.current_phase(input.project)
+      input_status_code = current_phase&.prescreening_enabled ? 'prescreening' : 'proposed'
```

Comment:
As an admin, I think you can post outside the current phase. For example, I think you could create a future proposals phase (with prescreening) and then test creating a proposal. In that case, I think it would be desired to use `prescreening_enabled` from that future phase?

Reply:
Ok, I'll take a look at this. But it makes me wonder whether we might not, instead want to skip prescreening for admins/moderators in any case, as an alternative (and simpler) solution.

Reply:
I changed it by making sure the phase is passed to the initialization of the ParticipationMethod, in the controller. So now it should follow the behavior you described.

Reply:
This makes a lot of sense!